packages/openid4vc-client/src/OpenId4VcClientService.ts

```diff
@@ -205,7 +214,7 @@ export class OpenId4VcClientService {
   }
 
   public async requestCredential(agentContext: AgentContext, options: RequestCredentialOptions) {
-    const credentialFormat = 'ldp_vc'
+    const supportedCredentialFormats = ['ldp_vc']
```

Comment:
Should this be configured by the user of this module? Might be useful to specify when doing a call of in the module configuration. Open for suggestions.

Reply:
I think we should set the default to all supported, and the user can overwrite it.

Reply:
I'm probably going to take over some of the things in this PR, as I'm going to work on adding JWT VC support to the openid client now